src/pages/ThorChainLP/components/LpType.tsx

```diff
@@ -139,12 +181,105 @@ export const LpType = ({
     setValue(fromOpportunityId(opportunityId).opportunityType)
   }, [opportunityId, setValue])
 
+  const subtitleStyle = useMemo(() => {
+    return {
+      '& svg': {
+        display: 'inline-block',
+        verticalAlign: 'middle',
+      },
+    }
+  }, [])
+
+  const informationsByPosition = useMemo(() => {
+    const displayAmounts = (position: AsymSide | 'sym') => ({
+      text: amountsByPosition?.[position] ? (
+        <>
+          <Amount.Crypto
+            value={amountsByPosition?.[position].underlyingAssetAmountCryptoPrecision}
+            symbol={asset?.symbol ?? ''}
+            maximumFractionDigits={6}
+          />
+          <Amount.Crypto
+            value={amountsByPosition?.[position].underlyingRuneAmountCryptoPrecision}
+            symbol={thorchainAsset?.symbol ?? ''}
+            maximumFractionDigits={6}
+          />
+        </>
+      ) : (
+        <>
+          <Skeleton mb={1} height='14px' width='100%' />
+          <Skeleton height='14px' width='100%' />
+        </>
+      ),
+      props: {
+        mt: !amountsByPosition?.[position] ? 2 : 1,
+      },
+    })
+
+    const newPosition = {
+      text: translate('pools.newPosition'),
+      icon: <CircleCrossIcon />,
+      tooltipText: translate('pools.newPositionTooltip'),
+    }
+    const notSupported = {
+      text: translate('pools.notSupported'),
+      icon: <CircleXIcon />,
+      props: { color: 'text.error' },
+      tooltipText: translate('pools.notSupportedTooltip'),
+    }
+
+    const runeInformations: PositionInformations = (() => {
+      if (hasSymPosition) return displayAmounts('sym')
```

Comment:
This might actually be where the bug comes from where it is showing my asym balance in the RUNE sym spot (instead of new position).

I suspect we need to re-work this logic.

Reply:
No, the logic seems good, if you have a `sym` position and tries to add some rune, you will simply add it to the sym position, you can't open a rune position if you already have a `sym` position opened

We might be able to find a clearer way to display this but currently it's not a bug but a feature!